appinventor/appengine/src/com/google/appinventor/client/wizards/RequestNewProjectNameWizard.java

```diff
@@ -0,0 +1,139 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2009-2011 Google, All Rights reserved
+// Copyright 2011-2020 MIT, All rights reserved
```

Comment:
No need to have the Google copyright here since this is new code. MIT is sufficient (and should start at 2021).